Added Workspace and Folder text size

ajkuftic opened the pull request:
Added new options for changing worksapce and folder label text sizes.

ajkuftic commented on themes/d868eaf4-cefc-4f41-8f1c-1f603726d2fa/chrome.css:
```diff
@@ -1,40 +1,155 @@
-    font-size: 22px;
-  }
-}
+@-moz-document url-prefix("chrome:") {
```
Check and version updated.

mr-cheffy replied:
Why do `:root:has` when you can do, for example, `#theme-SuperPins[uc-show-separator="never"] ~ #zen-main-app-wrapper ...`

ajkuftic reviewed (commented):
These changes work for me.

ajkuftic commented:
It worked when i submitted it, but when I removed the moz-document check, I missed removing the curly braces after that. I just removed them and tested that it works now.
